pkg/operator/starter.go

```diff
@@ -0,0 +1,140 @@
+package operator
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/openshift/csi-driver-manila-operator/pkg/controllers/manila"
+	"github.com/openshift/csi-driver-manila-operator/pkg/controllers/secret"
+	"github.com/openshift/csi-driver-manila-operator/pkg/generated"
+	"github.com/openshift/csi-driver-manila-operator/pkg/util"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	operatorapi "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	csicontrollerset "github.com/openshift/library-go/pkg/operator/csi/csicontrollerset"
+	csidrivercontroller "github.com/openshift/library-go/pkg/operator/csi/csidrivercontroller"
+	goc "github.com/openshift/library-go/pkg/operator/genericoperatorclient"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	operandName  = "manila-csi-driver"
+	operatorName = "manila-csi-driver-operator"
```

Comment:
note: the operator namespace is in `util`, not sure if want to have them in the same place

Reply:
It's name of the operator, not name of the namespace. It's just coincidence they're the same :-)

Reply:
I meant to say: have the operator name and namespace consts in the same place. But that's not really important